scripts/publish.js

```diff
@@ -49,10 +52,28 @@ fs.writeJSONSync(
   { spaces: 2 }
 );
 
+fs.writeJSONSync(
+  solidStartDriverJsonPath,
```

Comment:
If we eventually use patch-package - we do not need this solid-start dep removal